snapcraft/_store.py

```diff
@@ -73,6 +75,84 @@ def logout():
     logger.info('Credentials cleared.')
 
 
+def _get_usable_secret_keys(query=None):
+    ctx = gpgme.Context()
+    for key in ctx.keylist(query, True):
+        if not key.subkeys or not key.uids or key.disabled or not key.can_sign:
+            continue
+        mainkey = key.subkeys[0]
+        if (mainkey.expired or mainkey.revoked or mainkey.disabled or
+                mainkey.invalid):
+            continue
+        # We're currently only interested in RSA >= 4096-bit.
+        if mainkey.pubkey_algo != gpgme.PK_RSA or mainkey.length < 4096:
+            continue
+        yield key
+
+
+def _display_fingerprint(fingerprint):
```

Comment:
Maybe display is not the right word here, because it's not displaying anything. What about _get_formatted_fingerprint or _format_fingerprint ?